### DISCUSSION TITLE
How to enable `Fortran` file icons in `nvim-tree`?

### DISCUSSION THREAD

**Discussion by EhsanAramide** [nvim-tree/nvim-tree.lua#2068] [Q&A]
I tested `*.f`, `*.f90` and `*.for` but none of them work.

**Comment by gegoune** [comment]
Icons are provided by https://github.com/nvim-tree/nvim-web-devicons/. Can't remember if Fortran icon is there, but it's easy to add if it's missing.

**Comment by EhsanAramide** [comment]
I opened an [issue](https://github.com/nvim-tree/nvim-web-devicons/issues/236). Actually, the wrong file extension has been set.

**Reply by EhsanAramide** [reply]
I created a [pull request](https://github.com/nvim-tree/nvim-web-devicons/pull/237), you can review it and respond me again.
